libs/utils/utils.js

```diff
@@ -112,7 +114,7 @@ function getEnv(conf) {
   const query = location.searchParams.get('env');
 
   if (query) return { ...ENVS[query], consumer: conf[query] };
-  if (host.includes('localhost:')) return { ...ENVS.local, consumer: conf.local };
+  if (host.includes('localhost:') || host.includes('localhost.adobe.com:')) return { ...ENVS.local, consumer: conf.local };
```

Comment:
Adding support for running Milo locally with https

Reply:
That would nicely align with the docs: https://milo.adobe.com/docs/engineering/running-milo-locally 
I'd suggest if this gets merged, we slightly change the docs afterwards
`You will then be able to go to https://localhost.adobe.com:3000/` -> `You will then be able to go to https://localhost.corp.adobe.com:3000/`

`localhost.corp.adobe.com` is also whitelisted for IMS and should make testing anything that requires IMS easier.

Reply:
Is your suggestion to check whether the host includes `localhost.corp.adobe.com` as well or just remove the host from the documentation?

Reply:
Edit the documentation to only include `localhost.corp.adobe.com` otherwise IMS will not work locally and ppl might be confused

Reply:
I will remove these changes for now and handle them separately afterwards.